targets/TARGET_Ambiq_Micro/TARGET_Apollo3/TARGET_SFE_ARTEMIS_ATP/PinNames.h

```diff
@@ -128,8 +131,8 @@ typedef enum
     NC = NC_VAL
 } PinName;
 
-#define STDIO_UART_TX USBTX
-#define STDIO_UART_RX USBRX
+#define STDIO_UART_TX CONSOLE_TX
```

Comment:
This define should probably move to PinNameAliases.h

Reply:
Leaving as is for now.